librz/analysis/esil_dfg.c

```diff
@@ -514,7 +514,7 @@ static bool edf_bf(RzAnalysisEsil *esil) {
 }
 
 static bool _edf_consume_2_set_reg(RzAnalysisEsil *esil, const bool use_origin) {
-	const char *op_string = esil->current_opstr;
+	const char *op_string = RZ_STRBUF_SAFEGET (&esil->current_opstr);
```

Comment:
I wonder why `RZ_STRBUF_SAFEGET` is even necessary...

```C
RZ_API char *rz_strbuf_get(RzStrBuf *sb) {
	rz_return_val_if_fail (sb, NULL);
	return sb->ptr ? sb->ptr : sb->buf;
}
```
the only case where this can return NULL is when `sb` itself is NULL.  I think a regular `rz_strbuf_get (&esil->current_opstr)` is enough in all these cases.